tests/build_provider_packages_dependencies.py

```diff
@@ -114,13 +114,14 @@ class ImportFinder(NodeVisitor):
     AST visitor that collects all imported names in its imports
     """
 
-    def __init__(self, filename):
+    def __init__(self, filename: str) -> None:
+        
         self.imports: List[str] = []
-        self.filename = filename
+        self.filename: str = filename
```

Comment:
I don’t think this is needed, Mypy is able to infer this from the argument type IIRC.